pom.xml

```diff
@@ -358,6 +368,10 @@
 
     <build>
 
+        <filters>
```

Comment:
maybe add a one-line comment here, what this paragraph does? Not obvious for all of the readers